src/entrypoint/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"regexp"
+)
+
+type Response struct {
```

Comment:
The type is not exported, so it is better to start it's name from lowercase. As well as other fields.